randovania/layout/preset_migration.py

```diff
@@ -802,6 +803,12 @@ def _migrate_v49(preset: dict) -> dict:
 
     return preset
 
+def _migrate_v50(preset: dict) -> dict:
+    if preset["game"] == "dread":
+        preset["configuration"]["raven_beak_damage_table_handling"] = DreadRavenBeakDamageMode.CONSISTENT_LOW
```

Comment:
use pure string here instead of depending on any rdv API.
Also the enum is wrong here, curious why any test didn't fail.